deepchem/feat/tests/test_materials_featurizers.py

```diff
@@ -0,0 +1,66 @@
+"""
+Test featurizers for inorganic crystals.
+"""
+import numpy as np
+import unittest
+
+from deepchem.feat.materials_featurizers import ChemicalFingerprint, SineCoulombMatrix, StructureGraphFeaturizer
+
+
+class TestMaterialFeaturizers(unittest.TestCase):
+  """
+  Test material featurizers.
+  """
+
+  def setUp(self):
+    """
+    Set up tests.
+    """
+    self.formula = 'MoS2'
+    self.struct_dict = {
+        '@module':
+        'pymatgen.core.structure',
+        '@class':
+        'Structure',
+        'charge':
+        None,
+        'lattice': {
+            'matrix': [[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]],
+            'a': 1.0,
+            'b': 1.0,
+            'c': 1.0,
+            'alpha': 90.0,
+            'beta': 90.0,
+            'gamma': 90.0,
+            'volume': 1.0
+        },
+        'sites': [{
+            'species': [{
+                'element': 'Fe',
+                'occu': 1
+            }],
+            'abc': [0.0, 0.0, 0.0],
+            'xyz': [0.0, 0.0, 0.0],
+            'label': 'Fe',
+            'properties': {}
+        }]
+    }
+
+  def testCF(self):
+    """
+    Test CF featurizer.
+    """
+
+    featurizer = ChemicalFingerprint(data_source='matminer')
+
```

Comment:
A couple of good tests to add here might be the following:

1) A shape test, that checks the featurizer produces outputs of the right shape
2) A output sanity check, that checks that the output satisfies some sensible properties for this featurization